pipelines/datasets/gobierto_etl_esplugues_contracts/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+email = "popu-servers+jenkins@populate.tools "
+pipeline {
+    agent any
+    environment {
+        FOLDER = "gobierto_etl_esplugues_contracts"
+        PATH = "$HOME/.rbenv/shims:$PATH"
+        ETL_UTILS = "/var/www/gobierto-etl-utils/current/"
+        DATASET_NAME = "Contratos"
+        DATASET_SLUG = "contratos"
+        DATASET_TABLE_NAME = "${FOLDER}"
```

Comment:
Don't you prefer simpler table names such as tenders or contracts?

Reply:
> Don't you prefer simpler table names such as tenders or contracts?

I misunderstood you when you said:

> And replace it by a more specific name such as gobierto_etl_esplugues_contracts....

I'll change it

Reply:
Sorry, I meant just to simplify the table names, the name of the folder is ok

Reply:
Ohhh I seee now how's in Getafe ETL. Your initial proposal was fine, sorry! Could you leave it as it was in your first commit? :(

Reply:
Everything as it was in the initial commit? folder name too?